docs/models.yml

```diff
@@ -92,16 +92,23 @@ organization:
     type: boolean
     restriction_mode: B
 
-
   # Configuration (only for the server owner)
   enable_electronic_voting:
     type: boolean
     restriction_mode: B
+  limit_of_meetings:
```

Comment:
One question here: Is there a way to disable the limiting (Would be good to have!)? Whats about the values 0, -1, ... here. I think you should set a minimum.

Reply:
> One question here: Is there a way to disable the limiting (Would be good to have!)? Whats about the values 0, -1, ... here. I think you should set a minimum.

Values below 1 disable the limit, see last point in [impact on backend service](https://github.com/OpenSlides/OpenSlides/wiki/MeetingArchived#impact-on-backend-service)

Reply:
Please set a minimum of -1